### PR TITLE
Move metacity settings into anaconda.

### PR DESCRIPTION
**Pull request by dashea**
This is adapted from 1b4c5b3930cfc5db4071b0205859fb7bd2334331 and
f8ba2025765dd41b07b7e938d6cda73458c19cd5.

Install the metacity configuration files into a directory under
/usr/share/anaconda so taht they can be used by downstream projects.
Override $XDG_DATA_DIRS when running metacity to use our modified
settings.

The settings themselves are from the files currently installed by lorax.
The keybindings schema overrides disables a bunch of keyboard shortcuts,
and the theme file is a modification of the Adwaita theme that sets
has_title=False for maximized windows to hide the titlebar. The theme
file has been renamed to 'Anaconda' in order to co-exist with the
Adwaita theme installed with metacity, and wm.preferences override is a
new file to tell metacity to use the Anaconda theme.

Resolves: rhbz#1324889

### CODE REVIEW

**Comment by dashea** [comment]
Removed one too many lines from %files. Thanks, jenkins.

**Comment by jkonecny12** [comment]
> so taht they can be used by downstream projects.

^^^^
Please fix typo in commit message.

File: data/window-manager/Makefile.am [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# data/window-manager/Makefile.am for anaconda
+#
+# Copyright (C) 2015  Red Hat, Inc.
```
Comment (jkonecny12) [inline data/window-manager/Makefile.am]: Fix the Copyright year in the Makefiles.

**Comment by jkonecny12** [comment]
AFAICT this looks good to me otherwise.

**Comment by M4rtinK** [comment]
Also looks good to me & thanks for doing the cherry-pick/rebase! :)

**Comment by dashea** [comment]
Pushed.

**Pull request closed without merging**